docs/getting_started/default_images.md

```diff
@@ -0,0 +1,116 @@
+# Building
+
+- [Tutorial: Build a Default VHD or VHDX](#tutorial-build-a-default-vhd-or-vhdx)
+  - [Build VHD or VHDX](#build-vhd-or-vhdx)
+  - [Use Hyper-V to Boot Your Image](#use-hyper-v-to-boot-your-image)
+- [Tutorial: Build the default ISO](#tutorial-build-the-default-iso)
+
+## Tutorial: Build a Default VHD or VHDX
+
+The toolkit ships with several image configurations. These can be build using the tools outside the core repo.
+
+### Build VHD or VHDX
+
+Choose an image to build by invoking one of the following build commands from the _CBL-MarinerTutorials/toolkit_ folder.
+
+```bash
+# VHDX
+sudo make image CONFIG_FILE=./imageconfigs/core-efi.json 
+# VHD
+sudo make image CONFIG_FILE=./imageconfigs/core-legacy.json
+```
+
+The first time make image is invoked the toolkit downloads the necessary toolchain packages from the CBL-Mariner repository at packages.microsoft.com.  These toolchain packages are the standard set needed to build any local packages contained in the CBL-MarinerTutorials repo.  Once the toolchain is ready, make automatically proceeds to build any local packages.  In this case, the core repo's image configs do not use any of the packages located in the Tutorial repo so nothing will be built. Make will then assemble the packages gathered from the package server to build an image.
+The resulting images are placed in the CBL-MarinerTutorials/out folder
+
+> VHDX:       `CBL-MarinerTutorials/out/images/core-efi/`
+> VHD:        `CBL-MarinerTutorials/out/images/core-legacy/`
+
+### Use Hyper-V to Boot Your Image
```

Comment:
Consider refactoring this into a separate page since we reference these instructions in multiple places